doc/source/user_guide/templating.rst

```diff
@@ -1,24 +1,23 @@
 How to Add a New Template
 =========================
 
-You can easily add new templates to ``ansys-templates``. Before adding a new
-template, it is important that you read the CONTRIBUTING section.
+You can easily add new PyAnsys templates. However, before doing so, it is important that
+you read the CONTRIBUTING section.
```

Comment:
Is this section yet to be added or will it be to the overall Contributing section in the PyAnsys Developer's Guide. I assume that it will someday have a contributing.rst file of its own that references the Contributing topic in the PyAnsys Developer's Guide. At that point, we should have the link text here be Contributing--not CONTRIBUTING.

Reply:
Good catch on this one, @PipKat! I was planning to link this to some contributing section when available in the dev-guide. I guess that for the moment we can link this to https://dev.docs.pyansys.com/guidelines/dev_practices.html#contributing-through-github

Reply:
Linked this to https://github.com/pyansys/dev-guide/issues/64